### PR TITLE
docs: add samples for tagging

### PR DESCRIPTION
**Pull request by olavloite**
Adds samples for request tags and transaction tags.

### CODE REVIEW

File: samples/query-tag.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    "Query stats last hour for request tag 'app=cart,env=dev,action=update':"
+  );
+  stats.forEach(row => {
+    console.log(
+      `${row.REQUEST_TAG} ${row.AVG_LATENCY_SECONDS} ${row.AVG_CPU_SECONDS}`
+    );
+  });
+
+  await database.close();
+  // [END spanner_query_tags]
```
Comment (skuruppu) [inline samples/query-tag.js:56]: According to the new style, this block has to go in an async function. You can use https://github.com/googleapis/nodejs-spanner/blob/master/samples/index-create-storing.js as a reference.

File: samples/query-tag.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+}
```
Comment (skuruppu) [inline samples/query-tag.js:57]: We also need the following here:
```
process.on('unhandledRejection', err => {
  console.error(err.message);
  process.exitCode = 1;
});
```

File: samples/transaction-tag.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  // const projectId = 'my-project-id';
+  // const instanceId = 'my-instance';
+  // const databaseId = 'my-database';
+
+  // Creates a client
+  const spanner = new Spanner({
+    projectId: projectId,
+  });
+
+  // Gets a reference to a Cloud Spanner instance and database.
```
Comment (skuruppu) [inline samples/transaction-tag.js:35]: Same as above in this file.

File: samples/query-tag.js (hunk 1 above)
Comment (olavloite) [reply samples/query-tag.js]: I've updated the sample accordingly.

One small question: The https://github.com/googleapis/nodejs-spanner/blob/master/samples/index-create-storing.js sample has a `usage: node createStoringIndex <INSTANCE_ID> <DATABASE_ID> <PROJECT_ID>` entry. Executing that command won't work (as far as I can tell), as `createStoringIndex` is not a file. Shouldn't that be `node index-create-storing.js ...`?

File: samples/query-tag.js (hunk 2 above)
Comment (olavloite) [reply samples/query-tag.js:63]: Done.

File: samples/transaction-tag.js (hunk 3 above)
Comment (olavloite) [reply samples/transaction-tag.js]: Done.

File: samples/query-tag.js (hunk 1 above)
Comment (skuruppu) [reply samples/query-tag.js]: Sorry for the late reply. You're absolutely right that it has to say `<file>.js` for it to work.

**Comment by olavloite** [comment]
@skuruppu I've updated the samples according to the most recent specs.

**Review by skuruppu — APPROVED** [review]

**Pull request merged**